### PR TITLE
DS-193 yarn zero-install

### PR DESCRIPTION
**Pull request by lucas-paulger-sonarsource**

### CODE REVIEW

**Comment by lucas-paulger-sonarsource** [comment]
I had to rebuild the cached docker image but the build is fast when no deps are changed
https://cirrus-ci.com/build/5016209096179712

but we still need to use a cache for `.yarn/unplugged` since it is created at install and we apparently need it, but it should be ignored from git
https://github.com/yarnpkg/berry/issues/454

I also wanted to look for examples of this in the wild, of course yarn is using this approach
https://github.com/yarnpkg/berry/tree/master/.yarn/cache

and some others
https://github.com/cosmos/cosmjs/tree/main/.yarn
https://github.com/EventStore/Design-System/tree/main/.yarn/cache

File: .gitignore [hunk 1]
```diff
@@ -7,8 +7,8 @@
 
 # Swap the comments on the following lines if you wish to use zero-installs
 # Documentation here: https://yarnpkg.com/features/zero-installs
```
Comment (jeremy-davis-sonarsource) [inline .gitignore:9]: We should fix the comment here, it's confusing now

**Review by jeremy-davis-sonarsource — APPROVED** [review]
Seems to be the way forward

**Review by gregaubert — APPROVED** [review]
Looks good!

**Pull request merged**